code/modules/hydroponics/gene_modder.dm

```diff
@@ -24,14 +28,14 @@
 	component_parts += new /obj/item/weapon/stock_parts/console_screen(src)
 	component_parts += new /obj/item/weapon/stock_parts/scanning_module(src)
 	component_parts += new /obj/item/weapon/stock_parts/micro_laser(src)
-	component_parts += new /obj/item/weapon/stock_parts/micro_laser(src)
 	component_parts += new /obj/item/weapon/stock_parts/manipulator(src)
 	RefreshParts()
 
 /obj/machinery/plantgenes/RefreshParts()
 	rating = 0
 	for(var/obj/item/weapon/stock_parts/S in component_parts)
 		rating += S.rating-1
+	max_extract_pot = initial(max_extract_pot) + rating*5
```

Comment:
Will max out at 80. Your stock parts change broke this.

Add a scanner component if you want, for a a total of 3 rated parts.